restli-int-test-api/src/main/pegasus/com/linkedin/restli/examples/defaults/api/FooDefault.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.restli.examples.defaults.api
+
+import com.linkedin.restli.common.CollectionMetadata
+
+record FooDefault {
```

Comment:
Add a doc to explain which test scenarios this schema is used

Reply:
Though this is a test model, calling it "foo" is not nice. 

Call it: `FieldWithDefaultRecord`  or `HasFieldsWithDefault` ?

Reply:
Thanks for the comment, end up using the existing default filling purpose schema